master/Master.hs

```diff
@@ -269,24 +273,6 @@ consistentHashing msg = do
 --  if acks == Nothing then return ()
 --  else sendDecisionToRingWithRetry decision (timeout * 2)
 
-processMessages :: ReaderT (MVar MasterState) IO ()
```

Comment:
the name `listen` makes more sense but we should make sure to make parallel change in worker (right now it also has functions called processMessages and sendResponses)